maat-orchestration/src/main/java/uk/gov/justice/laa/crime/orchestration/service/orchestration/MeansAssessmentOrchestrationService.java

```diff
@@ -89,6 +89,11 @@ private ApplicationDTO processCrownCourtProceedings(WorkflowRequest request) {
                 request.getApplicationDTO(),
                 request.getUserDTO(),
                 StoredProcedure.ASSESSMENT_POST_PROCESSING_PART_2);
+
+        int financialAssessmentId = request.getApplicationDTO().getAssessmentDTO().getFinancialAssessmentDTO().getId().intValue();
+        AssessmentSummaryDTO assessmentSummaryDTO = assessmentSummaryService.getSummary(maatCourtDataService.getFinancialAssessment(financialAssessmentId));
```

Comment:
Do we need to retrieve the financial assessment here? This happens after the assessment has been created/updated so can't we just use the `FinancialAssessmentDTO` in the application object?

Reply:
Please note that the FinancialAssessmentDTO in the application object is different from the FinancialAssessmentDTO returned from MAAT Court Data API, though they are named exactly the same - one is closely aligned with MAAT UI and the other represents the financial_assessments data. We need the later to populate the assessment summary.

Reply:
I've compared the two data structures and can't pinpoint any missing fields that aren't in the MAAT `FinancialAssessmentDTO` that are in the Court Data API version? I'm just wondering whether we can avoid another outbound call.

Reply:
The rt code was missing. Thats why had to do this

Reply:
Yes, but it is not there for FullAssessment